### PR TITLE
Fix minor documentation errors

### PR DESCRIPTION
**Pull request by Avokadoen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ and only communicate with state changes in "object values" (bindings), not widge
 
 ### Examples
 
-- [Calculator](https://github.com/MasterQ32/Dunstwolke/tree/master/dunstwolke/examples/calculactor)
+- [Calculator](https://github.com/Avokadoen/core/tree/master/src/examples)
```
Comment (ikskuh) [inline README.md]: This should be a relative link, and neither a link to my or your github account

File: docs/src/index.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ as a concept:
 
 ## Source Code
 
-The project is currently written in C++ and is available on GitHub:
+The project is written in Zig and is available on GitHub:
 
 https://github.com/MasterQ32/Dunstwolke
```
Comment (ikskuh) [inline docs/src/index.md]: This should be changed as well

**Review by ikskuh — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Avokadoen) [reply README.md]: Ooof, I guess I need more coffee. Sorry about that

**Comment by ikskuh** [comment]
Looks good to me!

**Pull request merged**